src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/default-transcripts-list-item/DefaultTranscriptListItem.tsx

```diff
@@ -48,6 +48,16 @@ export type DefaultTranscriptListItemProps = {
 
 // NOTE: the width of the middle column is the same as the width of GeneOverviewImage, i.e. 695px
 
+export const defaultTranscriptLabelMap = {
+  selected: {
+    label: 'Selected',
+    helpText:
+      'The selected transcript is a default single transcript per protein coding gene that is representative of biology, well-supported, expressed and highly conserved'
+  }
+};
+
+export const canonicalType = 'selected'; // TODO Change this to transcript.mane/plus etc when available
```

Comment:
To reviewers: see if you'd prefer this to be a separate component.

To Jyo: see if you can figure out why transcript images are at a different height from the text.